src/pagopa.ts

```diff
@@ -9,9 +9,12 @@ import {
   PatternString
 } from "./strings";
 
+export const MIN_AMOUNT_DIGITS = 2;
 export const MAX_AMOUNT_DIGITS = 10;
 export const CENTS_IN_ONE_EURO = 100;
-export const AmountInEuroCents = PatternString(`[0-9]{${MAX_AMOUNT_DIGITS}}`);
+export const AmountInEuroCents = PatternString(
+  `^[0-9]{${MIN_AMOUNT_DIGITS},${MAX_AMOUNT_DIGITS}}$`
```

Comment:
Other than adding the range (2-10), I added the `^ $` delimiters, since `PatternString`, under the hood, is using `match()`, which matches substrings in the original string (so, without the anchors, things like `X123X` were successful matches)